Katana/Plastic/Value.swift

```diff
@@ -145,3 +187,65 @@ public struct Value: Equatable {
     return lhs.scalable == rhs.scalable && lhs.fixed == rhs.fixed
   }
 }
+
+/**
+ This allows `Value` instances to be initialized by integer and floating point literals,
+ having the scalable part equal to the numeric literal, and the fixed part equal to 0.
+ */
+extension Value: ExpressibleByIntegerLiteral, ExpressibleByFloatLiteral {
```

Comment:
`but then why it is ok?` well, but in that case we were not deciding between two approaches, that was the most natural one. In the LiteralConvertible vs  + case we have two options, both makes sense, but, saying that 200 is a .fixed/scalable(200) is a natural extension to me. At the end of the day you can create a `Value` starting from an Int using Value.fixed(200), this means that a Value and an Int can express the same thing.
I'm not saying that is good to introduce that per se, and that cannot lead to confusion in certain cases but I'm saying that `Value` and `Int` are interchangeable concepts in this specific context.